Project.toml

```diff
@@ -17,8 +17,8 @@ TimerOutputs = "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"
 [compat]
 JSON = "0.21"
 Preferences = "1.2"
-TimerOutputs = "0.5"
 PrettyTables = "1.1"
+TimerOutputs = "0.5"
```

Comment:
Can you undo this line move? I'd prefer to keep the compat entries together.

Reply:
Sorry, I made the commit in a rush and I didn't checked that my Pkg.update made that change ...

I will make a clean commit from scratch with the changes and force push to have a clean history!